main.nf

```diff
@@ -32,17 +32,12 @@ include { PIPELINE_COMPLETION     } from './subworkflows/local/utils_nfcore_rang
 //
 workflow NFCORE_RANGELAND {
 
-    take:
-    samplesheet // channel: samplesheet read in from --input
-
     main:
 
     //
     // WORKFLOW: Run pipeline
     //
-    RANGELAND (
-        samplesheet
-    )
+    RANGELAND ()
 
     emit:
     multiqc_report = RANGELAND.out.multiqc_report // channel: /path/to/multiqc_report.html
```

Comment:
Removing this would cause an error in line 84, where the `PIPELINE_COMPLETION` subworkflow requires this output (`NFCORE_RANGELAND.out.multiqc_report`). All of this is part of the template, are you sure that I should remove that?

Reply:
Correction afte rslack discussion: the MQC report should be kept for software version reporting purposes, but you could stiull add additional output channels